book/src/frontends/zk-continuations.md

```diff
@@ -0,0 +1,114 @@
+# RISCV and ZK-Continuations
+
+ZK-continuations can be used to make proofs for unbounded execution traces of
+Rust programs, where the trace is split into many different `chunks`. A proof
+is computed for each chunk, and all proofs can be combined with
+recursion/aggregation until a single proof remains.
+
+For the details of how memory is handled in the ZK-continuations case please
+see [this](https://github.com/powdr-labs/powdr/issues/814).
+
+`powdr-rs` has experimental support to ZK-continuations, which can be used as follows.
+
+Let's use as example test `many chunks` from the `riscv` crate:
+
+```rust
+{{#include ../../../riscv/tests/riscv_data/many_chunks/src/lib.rs}}
```

Comment:
Hehe, actually the comments say that it tries not to access any memory, we should probably change it to a more realistic example in an upcoming PR!